lms/djangoapps/learner_dashboard/urls.py

```diff
@@ -0,0 +1,11 @@
+"""
+Learner's Dashboard urls
+"""
+
+from django.conf.urls import patterns, url
+from . import views
+
+urlpatterns = patterns(
+    '',
```

Comment:
You're using an old format for your URLconf here. The new syntax looks like [this](https://docs.djangoproject.com/en/1.9/topics/http/urls/#example).